pkg/manifests/manifests_test.go

```diff
@@ -7,6 +7,7 @@ import (
 	"time"
 
 	ingressv1alpha1 "github.com/openshift/cluster-ingress-operator/pkg/apis/ingress/v1alpha1"
+	"github.com/openshift/cluster-ingress-operator/pkg/manifests"
```

Comment:
This is an import cycle (self-import); can just delete it.

Reply:
Fixed